python/graphstorm/model/utils.py

```diff
@@ -393,6 +393,11 @@ def _exchange_node_id_mapping(rank, world_size, device,
     # move mapping into CPU
     return gather_list[0].to(th.device("cpu"))
 
+def load_dist_nid_map(node_id_mapping_file, ntypes):
+    """ Wrapper for load_dist_nid_map.
```

Comment:
Please update the doc string.
_load_dist_nid_map is an internal function. You can not use an internal function to explain a public function.